biosimulators_utils/omex_meta/utils.py

```diff
@@ -51,12 +51,16 @@ def build_omex_meta_file_for_model(model_filename,
         raise ValueError('Metadata could not be extracted from model `{}`:\n  {}'.format(
             model_filename, '\n'.join(errors).replace('\n', '\n  ')))
 
-    model = editor.get_xml()
+    model = None
+    try:
+        model = editor.get_xml()
+    except:
+        pass
     if not model:
         errors = ''.join([logger[i_message].get_message() for i_message in range(len(logger))])
         raise RuntimeError('Model `{}` could not be read:\n  {}'.format(
             model_filename, errors.replace('\n', '\n  ')))
-    with open(model_filename, 'w') as file:
+    with open(model_filename, 'w', encoding="utf-8") as file:
```

Comment:
I'm hesitant to assuming the encoding is utf-8. It would better to get the encoding from the model object if possible.

Reply:
That's a good point, though in this case wouldn't you need a method in pyomexmeta to tell you the encoding?  I suppose we could also use one of the encoding guessing python packages to guess for us.

I can say that it seems to me the COMBINE community has settled on UTF-8 as a standard, though there's certainly room for other groups to come in with something new.  But even then, I *think* a pyomex file would always still use the same encoding.

Reply:
lxml, for example can report the encoding. Since pyomexmeta doesn't seem to be able to do this, lets let this go.